tests/llmcompressor/transformers/kv_cache/configs/gptq_kv_cache/gptq_kv_cache.yaml

```diff
@@ -0,0 +1,2 @@
+cadence: "commit"
+test_type: "regression"
```

Comment:
why is this empty?

Reply:
the tests use the same model for kv-cache-gptq so its in the code itself.
Regardless, the model needs to be here?